### PR TITLE
MBS-8098: Allow release-group-level-rels in release lookup

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement MBS-8098

This is particularly useful to know if a release's release group is part of a release group series.

On top of https://github.com/metabrainz/musicbrainz-server/pull/2173

### CODE REVIEW

**Review by mwiencek — APPROVED** [review]

**Review by yvanzo — APPROVED** [review]
LGTM but needs rebase on `master` after merging the other pull request it depends on.

**Comment by reosarevok** [comment]
@brainzbot, retest this please

**Pull request merged**
